.gitlab-ci.yml

```diff
@@ -142,7 +142,7 @@ before_script:
       -e local_release_dir=${PWD}/downloads 
       -e etcd_deployment_type=${ETCD_DEPLOYMENT} 
       -e kubelet_deployment_type=${KUBELET_DEPLOYMENT} 
-      cluster.yml;
+      cluster.yml --skip-tags=download --limit=k8s-cluster:kube-master;
```

Comment:
Why do you skip downloads? Shouldn't they run pretty fast when the images are already there?